src/main/java/lesson11Training/TrainingAnalyzer.java

```diff
@@ -0,0 +1,47 @@
+package lesson11Training;
+
+public class TrainingAnalyzer {
+
+    private static final int DURATION_SCORE_WEIGHT = 1;
+    private static final int CALORIES_BURNT_SCORE_WEIGHT = 2;
+    private static final int AVERAGE_BPM_SCORE_WEIGHT = 3;
+
+    private final Training training;
+
+    public TrainingAnalyzer(Training training) {
+        this.training = training;
+    }
+
+    public double getWeightedAverageScore(){
+        return (double)(getDurationScore() * DURATION_SCORE_WEIGHT + getCaloriesBurntScore() * CALORIES_BURNT_SCORE_WEIGHT +
+                getAverageBpmScore() * AVERAGE_BPM_SCORE_WEIGHT) / 6;
```

Comment:
Ta 6 to tzn. magic number, tzn. liczba, której znaczenia musimy się domyślać.

Reply:
poprawione :)